### PR TITLE
Small 0.96.6 CUP Tweaks

### PR DESCRIPTION
**Pull request by Eogos**
| Q | A |
| --- | --- |
| Bug fix? | No |
| New feature? | No |
| Needs wipe? | No |
| Fixed issues | N/A |

### Description:

CUP Presets still had the Nemmera repair vehicle that I forgot to remove as well as had their towing tractors all over the place and sometimes even in two spots. This PR will fix those small issues.

### Content:
- [x] Fixed Towing Tractor location across all Presets
- [x] Removed Nemmera from all CUP Presets

### Successfully tested on:
- [x] Local MP
- [x] Dedicated MP

### CODE REVIEW

**Comment by Eogos** [comment]
> 
> 
> Add this tweak to the changelog accordingly.
> Mention yourself there as it was done in the changelog entries for 0.96.5

ah yes, could you please direct me to where that is? My blind ass can't find it.

**Comment by Wyqer** [comment]
Readme

**Comment by veteran29** [comment]
https://github.com/KillahPotatoes/KP-Liberation/tree/v0.96.6#0966-tbd :P

**Comment by Eogos** [comment]
ah that would explain my confusion, was looking for change log

File: README.md [hunk 1]
```diff
@@ -194,6 +194,8 @@ class Missions
 * Fixed: Potato 01 was created after server restart, even if there was one saved.
 * Fixed: Missing variable `stats_blufor_teamkills_by_players`. Also no separation between by players or not by players for teamkills anymore.
 * Fixed: Factory storages could disappear randomly on save load.
+* Fixed: Some Presets had CUP Towing Tractor in the wrong place or duplicated. [Eogos](https://github.com/Eogos)
```
Comment (Wyqer) [inline README.md]:
```suggestion
* Fixed: Some Presets had CUP Towing Tractor in the wrong place or duplicated. Thanks to [Eogos](https://github.com/Eogos)
```

File: README.md [hunk 2]
```diff
@@ -194,6 +194,8 @@ class Missions
...
+* Fixed: CUP Presets still had the Nemmera in the support vehicle section. [Eogos](https://github.com/Eogos)
```
Comment (Wyqer) [inline README.md]:
```suggestion
* Fixed: CUP Presets still had the Nemmera in the support vehicle section. Thanks to [Eogos](https://github.com/Eogos)
```

**Pull request merged**